### DISCUSSION TITLE
Is there a way to get this response on auth user?

### DISCUSSION THREAD

**Discussion by donPuerto** [supabase/supabase#6845] [General]
Hi, I wanted to put logic on this response at auth user field "email_confirmed_at", see below

![image](https://user-images.githubusercontent.com/9329005/168463030-d4603e51-e208-4850-89e0-af7fa3d1763e.png)

However, there is no existing field to be found. See below,
![image](https://user-images.githubusercontent.com/9329005/168463527-f7abfd0d-6ede-4b03-9de8-ae74a611ae69.png)

Any idea how to get email_confirmed_at at auth user?

**Comment by GaryAustin1** [comment]
If it gets filled in (not null) it gets returned, at least in my quick test.  So you can check on the "key" or "property" being in the user object.
